main/modes/mode_dance.c

```diff
@@ -71,6 +71,7 @@ void danceNone(uint32_t tElapsedUs, uint32_t arg, bool reset);
 /*============================================================================
  * Variables
  *==========================================================================*/
+const char ledDancesExitText[] = "Exit: Start + Select";
```

Comment:
Unless there's a size issue, the other two modes with reminder text say `"Start + Select to Exit"`

Reply:
I think I tried that and it was a bit tight. Let me double check though, I may have tried something slightly different...

Reply:
Oh yeah:
![image](https://user-images.githubusercontent.com/666431/199152591-79c34fb1-17cd-4503-9001-0e4ebf205032.png)
With no space:
![image](https://user-images.githubusercontent.com/666431/199152652-26024689-1721-4be3-a884-125422bc014c.png)

Reply:
Hmm...
![image](https://user-images.githubusercontent.com/666431/199152874-3c469259-e895-4d8e-aeff-48cd7dbffdbb.png)

![image](https://user-images.githubusercontent.com/666431/199152904-7b44f051-08ef-4198-bf02-e40a1dfbdcdc.png)

![image](https://user-images.githubusercontent.com/666431/199152936-d0714510-d6be-4e33-a42e-65a92934a152.png)